lib/jsCompiler.js

```diff
@@ -40,11 +41,13 @@ module.exports = function(app) {
 
   jsFiles.forEach(function(file) {
     (function(file) {
-      var split = file.split(':'),
-          altdest = split[1];
-      file = split[0];
-      if (file === '.' || file === '..' || file === 'Thumbs.db' || fs.lstatSync(path.normalize(app.get('jsPath') + file)).isDirectory()
-) {
+      if (usingWhitelist === true) {
+        var split = file.split(':'),
```

Comment:
better to declare these vars outside of the if statement, then define them inside the if statement.